src/diagnostic-channel-publishers/src/winston.pub.ts

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for details.
+import {channel, IModulePatcher, PatchFunction} from "diagnostic-channel";
+
+export interface IWinstonData {
+    message: string;
+    meta: any;
+    level: string;
+}
+
+// register a "filter" with each logger that publishes the data about to be logged
+const winstonPatchFunction: PatchFunction = (originalWinston) => {
+    const originalConfigure = originalWinston.Logger.prototype.configure;
```

Comment:
I was originally patching configure until I remembered that users could add filters ad-hoc at runtime directly to the `loggerInstance.filters` array, so I decided to just patch calls to log instead.